randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -1074,7 +1096,9 @@ Extra - asset_id: 105
   > Door to Hellway
```

Comment:
Hellway definitely feels like it should have damage requirements

Reply:
I don't know what would be put there. Its a little platforming. Movement doesn't seem right because there is no trick. Adding varying amounts of health just opens up the floodgates for different amounts of health for all types of movement.

Reply:
"Its a little platforming" is a bit of an understatement IMO.
Think about it like this: is this trivially to do if you're playing with only 1HP? I would say no. Movement absolutely fits here, as if you're doing this hitless, you're absolutely gonna play more carefully, maybe even with disabled HJ. 
We don't need varying amounts of health, but we should attempt to ensure that 1/low HP runs have proper limitations.

Reply:
If we add movement then this room can't be done trickless.

Reply:
Added.